test/test_consuldump.py

```diff
@@ -21,7 +21,7 @@
 import shutil
 import sys
 
-sys.path.append('../low-level/files/opt/seagate/sspl/bin/')
+sys.path.append('../low-level/files/opt/seagate/sspl/setup/')
```

Comment:
can be taken from sspl_constants.py? or os.path?

Reply:
It can be addressed by EOS-16240